### ISSUE TITLE
Uncaught TypeError: $ is not a function in admin on Django 2.0 (admin is unusable)

### ISSUE THREAD

**Issue by seroy** [django-fluent/django-fluent-contents#93]
In django 2.0 combining Media objects was changed and admin form media outputs in wrong order:
```

<link href="/static/fluent_contents/admin/cp_admin.css" type="text/css" media="screen" rel="stylesheet" />
<script type="text/javascript" src="/static/fluent_contents/admin/cp_admin.js"></script>
<script type="text/javascript" src="/static/fluent_contents/admin/cp_data.js"></script>
<script type="text/javascript" src="/static/fluent_contents/admin/cp_plugins.js"></script>
<script type="text/javascript" src="/static/admin/js/vendor/jquery/jquery.js"></script>
<script type="text/javascript" src="/static/admin/js/jquery.init.js"></script>
<script type="text/javascript" src="/static/admin/js/calendar.js"></script>
<script type="text/javascript" src="/static/admin/js/admin/DateTimeShortcuts.js"></script>
<script type="text/javascript" src="/static/admin/js/core.js"></script>
<script type="text/javascript" src="/static/admin/js/admin/RelatedObjectLookups.js"></script>
<script type="text/javascript" src="/static/admin/js/actions.js"></script>
<script type="text/javascript" src="/static/admin/js/urlify.js"></script>
<script type="text/javascript" src="/static/admin/js/prepopulate.js"></script>
<script type="text/javascript" src="/static/admin/js/vendor/xregexp/xregexp.js"></script>
```

**Comment by seroy** [comment]
@vdboor v2.0.1 really?! Admin is still unusable! Can you merge #92 or fix this bug in another way?
Please... (╥_╥)

**Comment by vdboor** [comment]
@seroy I'll look into this!
I'm not eager to merge this fix, since I think it might hint at a deeper problem instead.

v2.0.1 was a critical bugfix release for django-polymorphic 2.0.1 compatibility (which was eventually fixed in django-polymorphic 2.0.2).

**Comment by vdboor** [comment]
I've released v2.0.2 with the fix.
django-slug-preview was also updated.

The error didn't happen in the `PlaceholderEditorAdmin` in my test sites so I didn't notice it before. The `PlaceholderFieldAdmin` however, did have this issue.

**Comment by seroy** [comment]
WOW! Many thanks!
